File: frontend-api/src/main/java/no/nav/melosys/tjenester/gui/dto/BehandlingDto.java
package no.nav.melosys.tjenester.gui.dto;

public class BehandlingDto {

    private Long behandlingID;

    private BehandlingOppsummeringDto oppsummering;

    private SaksopplysningerDto saksopplysninger;

    private boolean redigerbart = false;

    public Long getBehandlingID() {
        return behandlingID;
    }

    public void setBehandlingID(Long behandlingID) {
        this.behandlingID = behandlingID;
    }

    public BehandlingOppsummeringDto getOppsummering() {
        return oppsummering;
    }

    public void setOppsummering(BehandlingOppsummeringDto oppsummering) {
        this.oppsummering = oppsummering;
    }

    public SaksopplysningerDto getSaksopplysninger() {
        return saksopplysninger;
    }

    public void setSaksopplysninger(SaksopplysningerDto saksopplysninger) {
        this.saksopplysninger = saksopplysninger;
    }
    public boolean isRedigerbart() {
        return redigerbart;
    }

    public void setRedigerbart(boolean redigerbart) {
        this.redigerbart = redigerbart;
    }

}

File: frontend-api/src/main/kotlin/no/nav/melosys/tjenester/gui/dto/BehandlingDto.kt
package no.nav.melosys.tjenester.gui.dto


data class BehandlingDto(
    val behandlingID: Long,
    val oppsummering: BehandlingOppsummeringDto,
    val saksopplysninger: SaksopplysningerDto,
    val redigerbart: Boolean = false,
    val tilordnetIdent: String? = null,
    val tilordnetNavn: String? = null,
    val tilordnetMeg: Boolean = false,
    val kanTildeles: Boolean = false,
    val tildelingTilgjengelig: Boolean = false,
)

